pkg/anchor/handler/credential/handler.go

```diff
@@ -139,6 +149,31 @@ func (h *AnchorEventHandler) HandleAnchorEvent(actor, anchorRef, source *url.URL
 }
 
 func (h *AnchorEventHandler) processAnchorEvent(anchorInfo *anchorInfo) error {
+	anchorLink := anchorInfo.anchorLink
```

Comment:
I am wondering if this code should have been in observer.go line 340 This is where we parse VC with checking signature.

Reply:
I added the checks in this spot since this is where anchor events from other domains are processed. The Observer also receives anchors from the local domain and these are assumed to be valid (since the local domain created it).